proposals/function-pointers.md

```diff
@@ -114,7 +121,9 @@ funcptr_return_modifier
 ```
 
 The `unmanaged` calling convention represents the default calling convention for native code on the current platform, and is encoded as winapi.
-All `calling_convention`s are contextual keywords when preceded by a `delegate*`.
+All `identifer`s in the `unmanaged` case of the `calling_convention_specifier` must start with `CallConv`, and come from the
+`System.Runtime.CompilerServices` namespace in the program's core library (defined as one library that references no dependencies and defines
```

Comment:
Identifiers cannot really come from a namespace. Perhaps we should say that the identifier must match the name of a type in that namespace instead?

Reply:
Instead of `come from` I suggest `bind to a type in`

---
In reply to: [422510659](https://github.com/dotnet/csharplang/pull/3441#discussion_r422510659) [](ancestors = 422510659)